ssl-config-core/src/main/scala/com/typesafe/sslconfig/ssl/FakeKeyStore.scala

```diff
@@ -21,64 +22,109 @@ import java.security.interfaces.RSAPublicKey
  */
 class FakeKeyStore(mkLogger: LoggerFactory) {
   private val logger = mkLogger(getClass)
-  val GeneratedKeyStore = "generated.keystore"
-  val DnName = "CN=localhost, OU=Unit Testing, O=Mavericks, L=Moon Base 1, ST=Cyberspace, C=CY"
-  val SignatureAlgorithmOID = AlgorithmId.sha256WithRSAEncryption_oid
+
+  val GeneratedKeyStore: String = fileInDevModeDir("generated.keystore")
+  val ExportedCert: String = fileInDevModeDir("service.crt")
+  val TrustedAlias = "playgeneratedtrusted"
+  val DistinguishedName = "CN=localhost, OU=Unit Testing, O=Mavericks, L=Play Base 1, ST=Cyberspace, C=CY"
   val SignatureAlgorithmName = "SHA256withRSA"
+  val SignatureAlgorithmOID: ObjectIdentifier = AlgorithmId.sha256WithRSAEncryption_oid
+
+  object CertificateAuthority {
+    val ExportedCertificate = "target/dev-mode/ca.crt"
```

Comment:
This is related to https://github.com/lightbend/ssl-config/pull/113#issuecomment-424679754